iotqatools/cb_v2_utils.py

```diff
@@ -348,7 +346,10 @@ def __create_attributes(self, entity_context):
 
     def create_entities(self, context, entities_number, attributes_number):
```

Comment:
Parameter 'context' should not be present. This function is based on the context and context may be different depending on the test.